pkg/cmd/root/root_test.go

```diff
@@ -0,0 +1,47 @@
+package root
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aerogear/charmil/validator/rules"
+	"github.com/redhat-developer/app-services-cli/internal/build"
+	"github.com/redhat-developer/app-services-cli/internal/config"
+	"github.com/redhat-developer/app-services-cli/internal/mockutil"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+	"github.com/redhat-developer/app-services-cli/pkg/localize/goi18n"
+)
+
+func Test_RootRhoasCommand(t *testing.T) {
```

Comment:
ValidateCommandsUsingCharmilValidator

Reply:
`Test_ValidateCommandsUsingCharmilValidator`
did this, to run the test